free-1.md

```diff
@@ -5,51 +5,73 @@
+
 ### Why you?
 
 _What excites you about this idea? How did you come up with it?_
 
+It excites me a lot because I am also someone who is interested in betting and do bet a lot. I feel like it brings an extra excitement to sit in front of a TV and watch any game. So, I was thinking on what I could improve in the area of sports betting, and I thought this was a problem that needs a solution.
+
 ### Domain
 
 _Describe the project's domain in five words._
 
+Facilitating and typing sports betting
+
 ### Interface (syntax)
 
 _How might the user interact with the language? What does programming look
 like? Why is this the right way to interact with the problem domain?_
 
+The user might get asked by the language on what game he would like to bet on, and then what type of bet in the game. The programming behind that would find the game and the bet the user wants, and it to his bet slip. I believe this is the right way to interact with the problem domain because it is much more direct as the user knows what he wants, and can easily create his bets from here without making a mistake.
+
 ### Operation (semantics)
 
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+The program can interact with the user in a way of question and answer. When the program runs, it will ask which game and what type of bet the user wants to bet on. It will then add that bet to the user's bet slip, and ask if he wants to add more games. In the end, the program will display the total odds of the bet slip, which will show how much the total payout is when a certain amount is placed.
+
+There are a few errors that can occur. One is that if the match the user wants to bet on does not exist in the system, the program might display an error message saying that game is not in the system, and would ask the user to pick a new game. Another error is if the type of bet does not exist. Obviously, not everything can be bet on. For example, if the user wants to bet on something like "Player X from Team Y will have his hair combed to the right", I am almost certain that would not be in the system, so the program would ask again until a valid bet is given. 
+
 ### Expressiveness
 
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+Making a sports bet should be easy in this language. If anyone is using this language, they should avoid the confusing numbers and terms of any betting site, and use the language to create the bet. Combining matches from different sports in the same bet should be difficult, but possible. Creating illegal bets (i.e choosing both teams to win the same game) should be impossible.
```

Comment:
good point you shouldn't allow two teams who are playing each other to both win!